### PR TITLE
New UI for comment section

### PR DESCRIPTION
**Pull request by SubhamB2003**
Issue Solved: #508

### CODE REVIEW

File: src/components/Post/CommentDialogBox.jsx [hunk 1]
```diff
@@ -29,86 +25,78 @@ const CommentDialogBox = ({
...
+    <Box sx={{
+      overflowY: "scroll", '&::-webkit-scrollbar': {
+        width: 0,
+      }
+    }} borderRadius="16px" maxHeight="30vh">
+      {comments.length ? (
+        <>
+          {comments.map((userComment) => (
+            <div key={userComment.id}>
+              <div style={{ padding: "0.4rem 1rem", marginTop: "0.3rem", marginBottom: "0.3rem", display: "flex", justifyContent: "space-between", background: "lightgray", borderRadius: "10px" }}>
```
Comment (narayan954) [inline src/components/Post/CommentDialogBox.jsx]: can we use a css class to store these many styles, it's not right to use inline styles

**Comment by narayan954** [comment]
Also, don't completely remove the earlier modal. Let it stay and instead of using it for view all comments, use that on single clicking the post picture/text

**Comment by SubhamB2003** [comment]
> Also, don't completely remove the earlier modal. Let it stay and instead of using it for view all comments, use that on single clicking the post picture/text


Hey, Can you tell me to add this comment UI only for a single post-view page?

**Comment by narayan954** [comment]
> ll me to add this comment UI only for a single post-view page?
> In this page only

No Im saying shift the early view functionality to click on post rather than view all comments. Let both exist. On clicking view all comments, you feature shows up and when single clicking the post, old view comes up

**Comment by SubhamB2003** [comment]
> 

@narayan954 Is it ok?

https://github.com/narayan954/dummygram/assets/94003231/4a1b0061-7f21-4f70-b1a4-7c9c6e24c21b

**Comment by narayan954** [comment]
> > 
> 
> @narayan954 Is it ok?
> 
>  dummaygram3.mp4

yes it is! have you commited this? so I can quickly review and merge

**Comment by SubhamB2003** [comment]
> > > 
> > 
> > 
> > @narayan954 Is it ok?
> > dummaygram3.mp4
> 
> yes it is! have you commited this? so I can quickly review and merge

Wait I commit now.

**Comment by narayan954** [comment]
> > > > 
> > > 
> > > 
> > > @narayan954 Is it ok?
> > > dummaygram3.mp4
> > 
> > 
> > yes it is! have you commited this? so I can quickly review and merge
> 
> Wait I commit now.

leave the commit. I've an idea. Just let it stay

**Comment by narayan954** [comment]
> > > > 
> > > 
> > > 
> > > @narayan954 Is it ok?
> > > dummaygram3.mp4
> > 
> > yes it is! have you commited this? so I can quickly review and merge
> 
> Wait I commit now.

you haven't commited yet ...

**Comment by SubhamB2003** [comment]
@narayan954 You can check now.

**Comment by narayan954** [comment]
> @narayan954 You can check now. I also fixed the 404 error page issue.

alright, on it! :)

**Comment by narayan954** [comment]
> > 
> 
> @narayan954 Is it ok?
> 
>  dummaygram3.mp4

you didn't push this?? old post view?
Also, comment section words are not checked for breaking 
image

**Comment by narayan954** [comment]
> > > 
> > 
> > 
> > @narayan954 Is it ok?
> > dummaygram3.mp4
> 
> you didn't push this?? old post view? Also, comment section words are not checked for breaking image

it's alright if you didn't do this, I'll update

**Comment by SubhamB2003** [comment]
> 

Yeah, I checked all of this it's working fine I only changed the single post view comment section because someone is already updated it.

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**